src/liboslexec/shadingsys.cpp

```diff
@@ -1024,6 +1024,7 @@ ShadingSystemImpl::ShadingSystemImpl(RendererServices* renderer,
     , m_opt_middleman(true)
     , m_opt_texture_handle(true)
     , m_opt_seed_bblock_aliases(true)
+    , m_opt_useparam(false)
```

Comment:
Will then be enabled by default once confidence in its correctness is established?